remix-lib/src/web3Provider/web3VmProvider.js

```diff
@@ -18,29 +18,29 @@ function web3VmProvider () {
   this.incr = 0
   this.eth = {}
   this.debug = {}
-  this.eth.getCode = function (address, cb) { return self.getCode(address, cb) }
-  this.eth.getTransaction = function (hash, cb) { return self.getTransaction(hash, cb) }
-  this.eth.getTransactionReceipt = function (hash, cb) { return self.getTransactionReceipt(hash, cb) }
-  this.eth.getTransactionFromBlock = function (blockNumber, txIndex, cb) { return self.getTransactionFromBlock(blockNumber, txIndex, cb) }
-  this.eth.getBlockNumber = function (cb) { return self.getBlockNumber(cb) }
-  this.debug.traceTransaction = function (hash, options, cb) { return self.traceTransaction(hash, options, cb) }
-  this.debug.storageRangeAt = function (blockNumber, txIndex, address, start, end, maxLength, cb) { return self.storageRangeAt(blockNumber, txIndex, address, start, end, maxLength, cb) }
-  this.debug.preimage = function (hashedKey, cb) { return self.preimage(hashedKey, cb) }
+  this.eth.getCode = (...args) => self.getCode(...args)
```

Comment:
I really don't understand why we do that ^^.
Would this work, o you have issue with the `this` ?
```typescript
this.eth = {
  getCode: this.getCode,
  getTransaction: this.getTransaction,
  ...
}
```